module1/homework/grayscaleImages/test.cpp

```diff
@@ -1,9 +1,9 @@
 #include <algorithm>
 #include <array>
-#include <utility>  // for std::pair<>
+#include <utility>
 #include <vector>
```

Comment:
You can remove array,utility and vector from there since there are in compression.hpp you have included here.